checks/frozen_deps.go

```diff
@@ -43,6 +112,7 @@ func filePredicate(name string, logf func(s string, f ...interface{})) bool {
 	case "package-lock.json", "npm-shrinkwrap.json":
 		logf("nodejs packages found: %s", name)
 		return true
+	// TODO: I don't think requirement allows pinning
```

Comment:
Replace `I don't think` -> `TODO(laurentsimon)` ?

Reply:
+1. can improve comment, also change it to say check for sha hashes in requirements.txt, maybe with this example - https://davidwalsh.name/hashin